Mathlib/Analysis/Convex/Radon.lean

```diff
@@ -22,14 +23,24 @@ convex hull, radon
 
 -/
 
-
 open Set Finset
 open BigOperators
 
 universe u
 
 variable {𝕜 : Type*} {E : Type u} [LinearOrderedField 𝕜] [AddCommGroup E] [Module 𝕜 E]
 
+lemma radon_partition_aux {ι : Type*} {z : ι → E} (w : ι → 𝕜) (I J : Finset ι) (h_disj : Disjoint I J)
+    (h_wsum_zero : ∑ i in I, w i + ∑ j in J, w j = 0) (h_wsum_I_pos : ∑ i in I, w i ≠ 0)
```

Comment:
You should not need this, as in that case both `∑ i in I, w i` and `∑ i in J, w i` are zero and `centerMass I w z = centerMass J w z = 0` by the junk value of division by zero (`a / 0 = 0`).
```suggestion
    (h_wsum_zero : ∑ i in I, w i + ∑ j in J, w j = 0)
```